draft-ietf-quic-transport.md

```diff
@@ -2891,6 +2947,34 @@ PATH_CHALLENGE frame previously sent by the endpoint, the endpoint MAY generate
 a connection error of type UNSOLICITED_PATH_RESPONSE.
 
 
+## NEW_TOKEN frame {#frame-new-token}
+
+An server sends a NEW_TOKEN frame (type=0x10) to provide the client a token to
```

Comment:
0x10 conflicts with possible values for the STREAM frame.

Reply:
Failure to totally update PR from @nibanks